docs: Explain a bit more (#254)
prisma/engines-wrapper: README.md

  </h2>
</div>
  

> This repository contains the code for packages `@prisma/engines`, `@prisma/engines-version`, `@prisma/fetch-engine` and `@prisma/get-platform`. They are wrapping the [Prisma Rust Engines](https://github.com/prisma/prisma-engines) in npm packages to use by the [Prisma CLI and Client](https://github.com/prisma/prisma) and other tooling. 

Reviewer: ```suggestion
This repository contains the code for packages `@prisma/engines`, `@prisma/engines-version`, `@prisma/fetch-engine` and `@prisma/get-platform`. They are wrapping the [Prisma Rust Engines](https://github.com/prisma/prisma-engines) in npm packages to be used by the [Prisma CLI and Client](https://github.com/prisma/prisma) and other tooling. 
```

> [The automated publish pipeline](https://github.com/prisma/engines-wrapper/actions/workflows/publish-engines.yml), triggered itself during the publish process of [`prisma-engines`](https://github.com/prisma/prisma-engines), also triggers use of these packages via GitHub Actions workflows in other repositories (e.g. [`prisma/prisma`](https://github.com/prisma/prisma) and [`prisma/prisma-fmt-wasm`](https://github.com/prisma/prisma-fmt-wasm)).

Reviewer: ```suggestion
[The automated publish pipeline](https://github.com/prisma/engines-wrapper/actions/workflows/publish-engines.yml) triggers itself during the publish process of [`prisma-engines`](https://github.com/prisma/prisma-engines). It also triggers version updates via GitHub Actions workflows in dependent repositories (e.g. [`prisma/prisma`](https://github.com/prisma/prisma) and [`prisma/prisma-fmt-wasm`](https://github.com/prisma/prisma-fmt-wasm)).
```

Author: 1. The pipelines is triggered via prisma-engines
2. prisma-fmt-wasm technically is not a version update :/

Reviewer: Sorry, by bad! Ignore this then.

> [The automated publish pipeline](https://github.com/prisma/engines-wrapper/actions/workflows/publish-engines.yml), triggered itself during the publish process of [`prisma-engines`](https://github.com/prisma/prisma-engines), also triggers use of these packages via GitHub Actions workflows in other repositories (e.g. [`prisma/prisma`](https://github.com/prisma/prisma) and [`prisma/prisma-fmt-wasm`](https://github.com/prisma/prisma-fmt-wasm)).

Reviewer 2: ```suggestion
[The automated publish pipeline](https://github.com/prisma/engines-wrapper/actions/workflows/publish-engines.yml), is triggered during the publish process of [`prisma-engines`](https://github.com/prisma/prisma-engines) by the [engineer CLI](https://github.com/prisma/engineer/blob/master/src/trigger/mod.rs).
This `engines-wrapper` Action triggers use of these published npm packages via GitHub Actions workflows in other repositories (e.g. [`prisma/prisma`](https://github.com/prisma/prisma) and [`prisma/prisma-fmt-wasm`](https://github.com/prisma/prisma-fmt-wasm)).
```
